### PR TITLE
test(e2e-tests): supporting documents feature (FF+GC)

### PR DESCRIPTION
**Pull request by Fenisay**
## Ticket Number

UCD-829

## Description of change

Feature file + Glue codes
## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [ ] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

**Review by a6software — DISMISSED** [review]

File: e2e-tests/cypress/integration/appellant-submission-supporting-documents.feature [hunk 1]
```diff
@@ -0,0 +1,76 @@
+@wip
+Feature: Appellant submission - Supporting documents
+
+  An appeal must include one or many uploaded supporting documents.
```
Comment (frank-dodion) [inline e2e-tests/cypress/integration/appellant-submission-supporting-documents.feature:4]: Is this true? I thought an appeal could have zero supporting docs in this section.

Comment (Fenisay) [reply e2e-tests/cypress/integration/appellant-submission-supporting-documents.feature:4]: You're right 'could' is a better word, failed copy paste :/

**Comment by Fenisay** [comment]
Should I add the AC even if they are partially wrong and too abstract to match to only one scenario ?

**Review by a6software — APPROVED** [review]
LGTM

**Review by danWatkinson — APPROVED** [review]
happy that the features+glue describe the things that need to be described.

open question on some of the scenario titles- have raised https://pins-ds.atlassian.net/browse/UCD-1492 to review so that we can merge as-is and progress other work.

File: e2e-tests/cypress/fixtures/appeal-statement-invalid-wrong-type.mp3 [hunk 2]
```diff
@@ -0,0 +1 @@
+some,words,for,a,dummy,test,file
```
Comment (a6software) [inline e2e-tests/cypress/fixtures/appeal-statement-invalid-wrong-type.mp3:1]: I'm a big enough nerd to have checked whether a text file with a `.mp3` extension is sufficient to fool the mime type checker, and it is. 

:exploding_head:

File: e2e-tests/cypress/support/appellant-submission-supporting-documents/confirmNumberSupportingDocumentsAccepted.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+module.exports = (numberAccepted) => {
+  cy.goToSupportingDocumentsPage();
+
+  if (numberAccepted) {
+    cy.get('.moj-multi-file-upload__list').within(() => {
+      cy.get('.govuk-summary-list__row').should('have.length', numberAccepted);
```
Comment (a6software) [inline e2e-tests/cypress/support/appellant-submission-supporting-documents/confirmNumberSupportingDocumentsAccepted.js:6]: nice.

**Comment by a6software** [comment]
Ahh it was merged before I concluded my review. Fortunately, I was about to hit the approve button. 

:+1:

**Pull request merged**